File: W3/Task 1 - Python/person.py

class Person:

    def __init__(self, fname: str, lname: str, age: int) -> None:
        self.__age: int = age
        self.first_name: str = fname
        self.last_name: str = lname
        return None
   
    def getAge(self) -> int:
        return self.__age

    def ageUp(self) -> None:
        self.__age = self.__age + 1

    def getFullname(self) -> str:
        return f"{self.first_name} {self.last_name}"

    def printFullname(self) -> None:
        print(f"{self.first_name} {self.last_name}")

File: W3/Task 2- Javascript/person.js
class Person {
    #age;
    constructor(first_name, last_name, age){
        this.first_name = first_name;
        this.last_name = last_name;
        this.#age = age;
    }
    getAge(){
        return this.#age;
    }
    ageUp(){
        this.#age++;
    }
    getFullname(){
        return (`${this.first_name} ${this.last_name}`);
    }
    printFullName(){
        console.log(this.getFullname());
    }
}

module.exports = Person